pkg/resource/certificate_authority_activation/hooks.go

```diff
@@ -16,13 +16,40 @@ package certificate_authority_activation
 import (
 	"context"
 
+	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
 	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
 	ackerr "github.com/aws-controllers-k8s/runtime/pkg/errors"
 	ackrtlog "github.com/aws-controllers-k8s/runtime/pkg/runtime/log"
 	svcsdk "github.com/aws/aws-sdk-go/service/acmpca"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func (rm *resourceManager) writeCertificateChainToSecret(
+	ctx context.Context,
+	certificate string,
+	certificateChain string,
+	resourceNamespace string,
+	secretKeyReference *ackv1alpha1.SecretKeyReference,
+) (err error) {
```

Comment:
Minor nit:

```suggestion
) error {
```

A named return value is not required here.